src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
```

Comment:
```suggestion
  const [gameHistory, setGameHistory] = useState<string[][]>([initialBoard])
```

history almost feels like a reserved word in webdev since window.history/other routing solutions are used so much. I'd avoid naming anything history unless it for routing.